engine/src/rocksdb_engine.rs

```diff
@@ -54,6 +54,10 @@ impl From<RocksError> for EngineError {
 pub struct RocksEngine {
     /// The inner storage engine of `RocksDB`
     inner: Arc<DB>,
+    /// The tables of current engine
+    tables: Vec<String>,
+    /// The size cache of the engine
+    size: Arc<AtomicU64>,
```

Comment:
It seems there's no need for `Arc` here.

Reply:
Removed